### PR TITLE
document circuit relay in docs/experimental-features

### PR DESCRIPTION
**Pull request by vyzo**

### CODE REVIEW

**Comment by vyzo** [comment]
Summoning @whyrusleeping @Kubuxu

File: docs/experimental-features.md [hunk 1]
```diff
@@ -212,3 +213,55 @@ Basic usage:
...
+## Circuit Relay
+
+Allows peers to connect through an intermediate relay node when there
+is no direct connectivity.
+
+### State
+Experimental
+
+### In Version
+master
```
Comment (magik6k) [inline docs/experimental-features.md]: It's safe to say it will be in `0.4.11` here.

Comment (vyzo) [inline docs/experimental-features.md]: cool, will add.

File: docs/experimental-features.md [hunk 2]
```diff
@@ -212,3 +213,55 @@ Basic usage:
...
+try to dial through known relays:
+`ipfs swarm connect /p2p-circuit/ipfs/QmB`
+
+Peers can see their (unspecific) relay address in the output of
+`ipfs swarm addrs listen`
+
+### Road to being a real feature
+
+- [ ] Needs more people to use it and report on how well it works.
+- [ ] Advertise relay addresses to the DHT for NATed or otherwise
```
Comment (Stebalien) [inline docs/experimental-features.md]: I've opened an issue to discuss this: https://github.com/libp2p/specs/issues/33

**Comment by MichaelMure** [comment]
After reading this, I still have a question. Once you have configured a relay node, would that mean that all the traffic would go through this node (as a proxy) or would that be a fallback if no better route is found ?

Also, would protocol labs provide public relay enabled node ? @lgierth ?

**Comment by whyrusleeping** [comment]
Yes, we will provide public relay enabled nodes.

**Comment by vyzo** [comment]
> Once you have configured a relay node, would that mean that all the traffic would go through this node (as a proxy) or would that be a fallback if no better route is found ?

For initial (experimental) deployment, it's manual -- you have to manually open connections to peers using a relay.

**Pull request merged**